src/pylibsshext/channel.pyx

```diff
@@ -166,19 +167,23 @@ cdef class Channel:
             raise LibsshChannelException("Failed to execute command [{0}]: [{1}]".format(command, rc))
         result = CompletedProcess(args=command, returncode=-1, stdout=b'', stderr=b'')
 
-        cdef callbacks.ssh_channel_callbacks_struct cb
-        memset(&cb, 0, sizeof(cb))
+        cb_size = sizeof(callbacks.ssh_channel_callbacks_struct)
+        cdef callbacks.ssh_channel_callbacks_struct *cb = <callbacks.ssh_channel_callbacks_struct *>PyMem_Malloc(cb_size)
+        if cb is NULL:
+            raise LibsshChannelException("Memory allocation error")
```

Comment:
Codecov says this line is uncovered: https://app.codecov.io/gh/ansible/pylibssh/pull/658#17ffa324129dc304109f4a66c79769d7-R173.

Could you add a test covering this newly added line so that all the new lines in the patch are covered?

Reply:
Yep. Codecov shows that the lines that this patch touches are all covered.